spring-cloud-gcp-storage/src/main/resources/META-INF/spring/aot.factories

```diff
@@ -0,0 +1,2 @@
+org.springframework.aot.hint.RuntimeHintsRegistrar=\
+com.google.cloud.spring.storage.integration.aot.StorageIntegrationRuntimeHint
```

Comment:
Did we decide to replace this with the dynamic annotation alternative?

Reply:
Good point. Done.